### DISCUSSION TITLE
why aspire don't show catch block message in standard log report

### DISCUSSION THREAD

**Discussion by AjayShaktawat** [microsoft/aspire#5331] [Q&A] [answered]
Every developer use try catch block to handle error but aspire don't show catch block message in log report. If we not use try catch block 
aspire show error when we use try catch block to handle error aspire not show any error.

**Comment by davidfowl** [comment, marked as answer]
The dashboard is showing logs from the application, not exceptions. If your application does not log the exception then it will not show up in the dashboard.
